README.md

```diff
@@ -56,4 +56,4 @@ the project and submit pull requests.
 
 The library is available under [MIT license](https://github.com/ionide/dotnet-proj-info/blob/master/LICENSE.md), which allows modification and redistribution for both commercial and non-commercial purposes.
 
-Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
\ No newline at end of file
+Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
```

Comment:
Not sure what's different here - newline?
I used GitHub UI to create the fork, branch and PR, haven't touched the file myself.